util/fixtures/makeWorkorder.js

```diff
@@ -15,6 +15,7 @@ module.exports = function makeWorkorder(userId, workflowId) {
     finishDate: "2029-02-23T00:00:00Z",
     finishTime: "2029-02-23T00:00:00Z",
     summary: "Do the work",
+    // TODO: for some reason 'startTimestamp' now shows up as 'finish date/time' in the portal
```

Comment:
@david-martin also this is a separate issue that could be looked at in the raincatcher demo itself (not sure if it's considered a bug or a feature)

cc @paolobueno